Makefile

```diff
@@ -53,6 +53,9 @@ view-submission: ${TEX_MAIN_SUBMISSION} ${IMAGES}
 
 clean:
 	latexmk -C
+	[ -d "_minted" ] && rm -r _minted || true
+	[ -d "_minted-$(TEX_MAIN_PAPER)" ] && rm -r "_minted-$(TEX_MAIN_PAPER)" || true
+	[ -d "_minted-$(TEX_MAIN_SUBMISSION)" ] && rm -r "_minted-$(TEX_MAIN_SUBMISSION)" || true
```

Comment:
Would `rm -rf` work here, to avoid having to do the existence check?

Note: I can imagine that there might be reasons we don't want the `-f` flag; I don't know the trade-offs by heart